code/modules/fallout/reagents/alcohol.dm

```diff
@@ -250,7 +250,7 @@
 	taste_description = "freedom"
 	glass_icon_state = "nukavictoryglass"
 	glass_name = "Nuka Victory"
-	glass_desc = "Nuka Cola with an <BIG>AMERICAN<BIG> twist."
+	glass_desc = "Nuka Cola with an AMERICAN twist."
```

Comment:
This makes the PR non-atomic (ha ha). 
Generally though, you don't want to be making multiple changes at once if the PR isn't meant for that. 
If this PR's stated intent is to remove the helmets, then it should only remove the helmets.
This ensures that PRs can be easily reverted without unexpected side effects.